pkg/controllers/netpol/policy.go

```diff
@@ -413,6 +413,13 @@ func (npc *NetworkPolicyController) processEgressRules(policy networkPolicyInfo,
 					return err
 				}
 			}
+			for _, portProtocol := range egressRule.namedPorts {
+				comment := "rule to ACCEPT traffic from source pods to all destinations selected by policy name: " +
+					policy.name + " namespace " + policy.namespace
+				if err := npc.appendRuleToPolicyChain(iptablesCmdHandler, policyChainName, comment, targetSourcePodIPSetName, "", portProtocol.protocol, portProtocol.port); err != nil {
```

Comment:
I missed this when I looked through it last. You'll need to remove `iptablesCmdHandler` here as that argument has been removed since the merge of #1010